content/events/2020-paris/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Paris
+date: '2020-10-29'
+end: '2020-10-30'
+location:
+  country: France
+  city: Paris
+  url: https://en.wikipedia.org/wiki/Paris
+  venue: "????"
```

Comment:
Hi folks! Can you please specify the venue?

Reply:
We are still in the process of confirming the venue and the official date of the event. I'll update the PR asap.